telegram/src/test/kotlin/com/github/kotlintelegrambot/network/apiclient/ApiClientIT.kt

```diff
@@ -13,6 +14,8 @@ abstract class ApiClientIT {
 
     protected lateinit var sut: ApiClient
 
+    protected val gson = Gson()
```

Comment:
I'm not really sure about this, I'd advocate for keeping this base test case as simple as possible and not start adding anything that is used by more than one test case here. 

A `Gson` instance is currently used in a couple of api client test cases only and there's not a great advantage from having it here.